services/metadata/backend/data/beol.json

```diff
@@ -599,8 +599,8 @@
             "url": {
                 "__type": "URL",
                 "type": "URL",
-                "url": "http://p3.snf.ch/project-166072",
-                "text": "http://p3.snf.ch/project-166072"
+                "url": "https://data.snf.ch/grants/grant/166072",
+                "text": "https://data.snf.ch/grants/grant/166072"
```

Comment:
generally (this goes for all metadata), it would be good to have more speaking texts here in the `text` property, rather than just copying the URL. The `text` property is what is displayed, so ideally it would be readable and nice-looking instead of just a URL